ui/components/app/snaps/snap-delineator/snap-delineator.js

```diff
@@ -83,7 +83,7 @@ export const SnapDelineator = ({
           />
           <Text
             variant={TextVariant.bodySm}
-            color={isError ? TextColor.textDefault : TextColor.textAlternative}
+            color={TextColor.Default}
```

Comment:
I'm just now realizing this is a change for the delineator. This would change it in a bunch of places.

Is that the intention @eriknson ?